### PR TITLE
Update search filter description

### PR DESCRIPTION
**Pull request by mclassic**
Update the search filter description to explain `date` parameter.

### CODE REVIEW

File: spec/paths/search.yaml [hunk 1]
```diff
@@ -14,7 +14,8 @@ get:
         - description: |
             The collection items filter requires a special format.
             Use "," for multiple allowed values.  Use ";" for multiple fields.
-            Use one or more of the listed filter fields. i.e. `is:customer email:bob@test.com`
+            Use one or more of the listed filter fields. i.e. `is:customer email:bob@test.com`.
+            The `date` value can be surrounded with quotes and accepts [any format that PHP's `strtotime` supports.](https://www.php.net/manual/en/function.strtotime.php)
```
Comment (2e3s) [inline spec/paths/search.yaml]: Can't the API be less technology-dependent?

Comment (2e3s) [inline spec/paths/search.yaml]: It's better to describe the particular usage cases which we will certainly support whatever technology is used for this endpoint(s).

Comment (mclassic) [reply spec/paths/search.yaml]: It will make the description field very long. Can you suggest where else I could put it?

Comment (2e3s) [reply spec/paths/search.yaml]: How long? Do users need a zillion use cases, everyone with that std library function provides? I think putting here is okay.

Comment (mclassic) [reply spec/paths/search.yaml]: Okay, I'll add some and get rid of this link.

**Comment by mclassic** [comment]
devs please review

**Pull request merged**